lib/customers.rb

```diff
@@ -0,0 +1,12 @@
+class Customers
```

Comment:
Would Customer maybe be a clearer name for the Class? Then when you're making an instance, it is one new customer (not a group).